web/src/components/SecureAdminConsole.jsx

```diff
@@ -34,9 +34,12 @@ class SecureAdminConsole extends React.Component {
         }
 
         const apps = await this.props.onLoginSuccess();
+        const pendingApp = await this.props.pendingApp();
```

Comment:
this function name is confusing, does indicate if it's a function or a variable
```suggestion
        const pendingApp = await this.props.getPendingApp();
```